craft_parts/parts.py

```diff
@@ -642,6 +639,23 @@ def part_has_overlay(data: Dict[str, Any]) -> bool:
     return spec.has_overlay
 
 
+def part_has_slices(data: Dict[str, Any]) -> bool:
+    """Whether the part described by ``data`` has slices in its stage-packages.
```

Comment:
How and when is this going to be accessed? It feels that it should be a part property instead of a call receiving a list of part names.

Reply:
This is needed for Rockcraft's [PR 464](https://github.com/canonical/rockcraft/pull/464).
It will work as a part property as well, and I don't have a particular preference for either. However, there is already a similar utility function called `part_has_overlay`, so I put this one in the same scope.